packages/story-editor/package.json

```diff
@@ -50,7 +50,6 @@
     "@googleforcreators/tracking": "*",
     "@googleforcreators/transform": "*",
     "@googleforcreators/units": "*",
-    "blurhash": "^1.1.4",
```

Comment:
@samwhale Maybe the reason you got that TypeError..?

Reply:
💭 That's a good idea. I don't think that is the cause though because I am adding this back in locally when trying to inline the module again

Reply:
How are you adding this back in? 🤔 

This package is needed in the editor, so it needs to be listed as a dependency.

Reply:
Reverted this change! It's only in the story-editor `package.json`, no other ones

Reply:
Trying stuff with these updates: [f35afad](https://github.com/GoogleForCreators/web-stories-wp/pull/5792/commits/f35afad43807d422006d1c5f6b0f4228c127ee37)